src/collections/_documentation/platforms/javascript/getting-started-dsn/react.md

```diff
@@ -1,42 +1,9 @@
-<!-- WIZARD -->
```

Comment:
i think this was copy paste - afaict it shouldnt be needed

Reply:
Confirmed, this doc doesn't appear to be referenced in platforms.yml